clients/nexus-client/src/lib.rs

```diff
@@ -5,8 +5,7 @@
 //! Interface for making API requests to the Oxide control plane at large
 //! from within the control plane
 
-use std::collections::HashMap;
-
+#[cfg(feature = "omicron-internal")]
```

Comment:
I'd maybe add a comment here talking about why this exists and reminding folks that for things not in onicron-common (eg uuid kinds) one should add a particular replace to both spots.

Reply:
Yes, definitely.  I ran out of steam on that part at the end of the day.